schemas/v1.0/azure.yaml.json

```diff
@@ -81,9 +81,9 @@
                         "ts"
                     ]
                 },
-                "moduleName": {
+                "module": {
                     "type": "string",
-                    "title": "Name of the module used to deploy the service",
+                    "title": "Path of the infrastructure module used to deploy the service relative to the root infra folder",
```

Comment:
I think we should make all relative paths in this file relative to the location of `azure.yaml`.  Right now I think it is not uniform.  This may lead to slightly longer paths but I think it will be easier to understand? Thoughts?

Reply:
Happy to defer this - but I think we should be thinking about this a bit more.

Reply:
The recent `docker` options are relative to the service as well.  Lets defer this for the time being and have a discussion.  

My opinion would be to keep the paths relative to the node that you are in since each service already requires the path to the project I would expect additional options in the service to be relative to that specified service path.